docs/_docs/configuration/markdown.md

```diff
@@ -42,6 +42,13 @@ currently supported options:
 * **transliterated_header_ids** - Transliterate the header text before generating the ID
 * **typographic_symbols** - Defines a mapping from typographical symbol to output characters
 
+### Example Usage
+```yaml
+markdown: kramdown
+kramdown:
+  html_to_native: true
+  ```
```

Comment:
The closing *triple backticks* should be aligned with the opening *triple backticks*